pkg/api/api.go

```diff
@@ -89,6 +89,12 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/a/:id/*", reqSignedIn, hs.Index) // App Root Page
 	r.Get("/a/:id", reqSignedIn, hs.Index)
 
+	//pubdash
+	if hs.Features.IsEnabled(featuremgmt.FlagPublicDashboards) {
+		r.Get("/p/:uid/:slug", redirectFromLegacyPanelEditURL, hs.Index)
+		r.Get("/p/:uid", redirectFromLegacyPanelEditURL, hs.Index)
```

Comment:
can we make this  `/public-dashboards/:uid` , the d for shorthand for dashboards is not very clear still maybe some can figure it out it stands for dashboards, but just a p will be even more cryptic

Reply:
Yep for sure. Once we connect this work with the backend work for public dashboards the urls will get changed

Reply:
Actually I'm gonna rename this now